r4/coverage-requirements-discovery/Package.md

```diff
@@ -0,0 +1,51 @@
+# CDS Hooks for Davinci's CRD
+
+# 1. Coverage Requirement Discovery
+
+This is a ballerina template for the insurance payers who are developing the solution for the Davinci’s Coverage Requirement Discovery. 
+
+This will contain the CDS hooks as service. The [documentation](https://build.fhir.org/ig/HL7/davinci-crd/hooks.html#hook-categories) states: “The **primary** hooks are `Appointment Book`, `Orders Sign`, and `Order Dispatch`. CRD Servers SHALL, at minimum, return a Coverage Information system action for these hooks, even if the response indicates that further information is needed or that the level of detail provided is insufficient to determine coverage. The **secondary** hooks are `Orders Select`, `Encounter Start`, and `Encounter Discharge`. These hooks MAY return cards or system actions, but are not expected to, and CRD clients are free to ignore any cards or actions returned.” 
+
+The following features are implmented,
+> 1. **CDS hooks as ballerina service**: All six hooks will be implemented as ballerina service. Since it is not mandatory to support all the hooks, the payer can define the relevant hooks for their services. `Config.bal` will contain all the hooks. 
+> 2. **Ballerina Records for CRD**: All of the types will be implemented as ballerina records to support type safety. `health.fhir.cds.records.bal` contains the implemented records.
```

Comment:
Since this is a Ballerina package, shall we link this to the package page in the Ballerina central?